src/Compilers/CSharp/Portable/Symbols/Source/SourceTypeParameterSymbol.cs

```diff
@@ -876,14 +866,8 @@ protected override TypeParameterBounds ResolveBounds(ConsList<TypeParameterSymbo
             }
             var map = _map.TypeMap;
             Debug.Assert(map != null);
-            // Use typeParameter.ConstraintTypesNoUseSiteDiagnostics rather than
```

Comment:
>// Use typeParameter.ConstraintTypesNoUseSiteDiagnostics rather than [](start = 12, length = 68)

Feels like this comment is still applicable/useful, no?

Reply:
> Feels like this comment is still applicable/useful, no?

GetConstraintTypesNoUseSiteDiagnostics is removed, ConstraintTypesNoUseSiteDiagnostics has no alternative

---
In reply to: [280534641](https://github.com/dotnet/roslyn/pull/35375#discussion_r280534641) [](ancestors = 280534641)